src/test/java/guitests/PanelRenameTest.java

```diff
@@ -134,4 +144,17 @@ public void panelRenameTest() {
         push(KeyCode.ENTER);
         sleep(EVENT_DELAY);
     }
+
+    @Test
+    public void panelRenameButton_onClick_panelGetsSelected() {
+        // Test for the currently selected panel when trying to rename
+
+        click(IdGenerator.getOcticonButtonIdReference(0, "renameButton"));
+        assertEquals(Optional.of(0), panels.getCurrentlySelectedPanel());
+        pushKeys(CREATE_RIGHT_PANEL);
+        click(IdGenerator.getOcticonButtonIdReference(1, "renameButton"));
+        assertEquals(Optional.of(1), panels.getCurrentlySelectedPanel());
+        pushKeys(CLOSE_PANEL);
+        pushKeys(KeyCode.ESCAPE);
```

Comment:
Seems like some UI tests are failing. If it continues to fail, adding another `sleep(EVENT_DELAY)` at the end of this test might help.